themes/default/static/programs/aws-simulated-server-interpolate-csharp/Program.cs

```diff
@@ -9,11 +9,12 @@
         Port = "8080",
     });
 
-    // Format takes a FormattableString and expands outputs correctly:
-    var url = Output.Format($"http://{webServer.HostName}:{webServer.Port}/");
+    // // Format takes a FormattableString and expands outputs correctly:
```

Comment:
I'm a bit of a stickler when it comes to checking in dead and/or commented code. 😇 (Always some exceptions of course, but rare, and generally accompanied by an additional comment explaining why the code is commented and not just deleted.) 

If this doesn't yet work and isn't referenced anywhere, I'd prefer we removed it and put it back when we're ready to use it.